### PR TITLE
[tdarr] new chart

### PR DESCRIPTION
**Pull request by auricom**
[Tdarr](url) chart based on the dependent 'common' chart.
Distributed Transcoding System

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [x] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/tdarr/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+appVersion: 2.00.07
+description: Tdarr is a self hosted web-app for automating media library transcode/remux management and making sure your files are exactly how you need them to be in terms of codecs/streams/containers etc.
+name: tdarr
+version: 1.0.0
+keywords:
+  - joplin
```
Comment (onedr0p) [inline charts/tdarr/Chart.yaml:7]: tags need updating

File: charts/tdarr/.helmignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+.vscode/
+*.tmproj
+# OWNERS file for Kubernetes
+OWNERS
```
Comment (onedr0p) [inline charts/tdarr/.helmignore:24]: also need *.gotmpl

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/tdarr/Chart.yaml (hunk 1 above)
Comment (auricom) [reply charts/tdarr/Chart.yaml]: Whoopsie, fixed!

**Comment by auricom** [comment]
Github action failed due to network issue, a manual trigger should be successful

**Review by onedr0p — APPROVED** [review]

**Pull request merged**